packages/react/src/Theme.ts

```diff
@@ -0,0 +1 @@
+export * from '@fluentui/react-internal/lib/Theme';
```

Comment:
Should we add an export from the index file for this?

Reply:
no need bc index of `react` does: `export * from '@fluentui/react-internal';`